packages/kit/src/runtime/client/remote.svelte.js

```diff
@@ -0,0 +1,840 @@
+/** @import { RemoteFormAction, RemoteQuery } from '@sveltejs/kit' */
+/** @import { RemoteFunctionResponse } from 'types' */
+
+import { app_dir } from '__sveltekit/paths';
+import { version } from '__sveltekit/environment';
+import * as devalue from 'devalue';
+import { DEV } from 'esm-env';
+import { HttpError, Redirect } from '@sveltejs/kit/internal';
+import {
+	app,
+	invalidateAll,
+	remote_responses,
+	started,
+	goto,
+	set_nearest_error_page,
+	query_map
+} from './client.js';
+import { create_remote_cache_key, stringify_remote_arg } from '../shared.js';
+import { tick } from 'svelte';
+
+// Initialize Cache API for prerender functions
+const CACHE_NAME = `sveltekit:${version}`;
+/** @type {Cache | undefined} */
+let prerender_cache;
+
+void (async () => {
+	if (!DEV && typeof caches !== 'undefined') {
+		try {
+			prerender_cache = await caches.open(CACHE_NAME);
+
+			// Clean up old cache versions
+			const cache_names = await caches.keys();
+			for (const cache_name of cache_names) {
+				if (cache_name.startsWith('sveltekit:') && cache_name !== CACHE_NAME) {
+					await caches.delete(cache_name);
+				}
+			}
+		} catch (error) {
+			console.warn('Failed to initialize SvelteKit cache:', error);
+		}
+	}
+})();
+
+/**
+ * @template T
+ * @implements {Partial<Promise<T>>}
+ */
+class Resource {
+	#init = true;
```

Comment:
😄 

```suggestion
	#init = false;
```

Reply:
oh wait there's already an `#inited`

Reply:
gonna go ahead and commit this, because it's impossible to test anything until we do